projects/fsastorefrontlib/src/assets/translations/index.ts

```diff
@@ -33,7 +43,12 @@ export const fstranslations: TranslationResources = {
   orderConfirmation,
   organization,
   orgUnit,
+  orgCostCenter,
```

Comment:
can we inject entire organizationTranslationChunksConfig instead of enlisting them one by one?